RELEASE.md

```diff
@@ -0,0 +1,63 @@
+# lnc-core Release Process
+
+This document describes the steps needed to release a new version of
+`@lightninglabs/lnc-core` and publish the package to the NPM registry.
+
+The steps below for Bumping proto versions and Versioning should be done in
+a PR with proper review.
+
+## Bumping proto versions
+
+When new versions of LND, Loop, Pool, Faraday, or Lightning Terminal are
+released, we should update the generated types in this package to match the
+latest versions.
+
+To do this, you need to update the `config` section in the
+[package.json](https://github.com/lightninglabs/lnc-core/blob/9d31f49dd9cacab1e7cb1d5664074fd554f42897/package.json#L7)
+file.
+
+```json
+  "config": {
+    "lnd_release_tag": "v0.16.0-beta",
+    "loop_release_tag": "v0.22.0-beta",
+    "pool_release_tag": "v0.6.2-beta",
+    "faraday_release_tag": "v0.2.9-alpha",
+    "lit_release_tag": "v0.9.0-alpha",
+    "protoc_version": "21.9"
```

Comment:
When should this be changed?

Reply:
Whenever it is necessary. There's no need to update it unless there's a specific reason to. Maybe an update to golang breaks this version of protoc, idk.